tests/scripts/all.sh

```diff
@@ -6396,6 +6396,12 @@ component_check_test_helpers () {
     python3 -m unittest tests/scripts/translate_ciphers.py 2>&1
 }
 
+component_test_pkgconfig () {
```

Comment:
Each component adds some CI runtime cost (it's an extra container instantiation). We have a lot of components because we test a lot of configurations (and compiler options, build systems, etc.). But we already have components that do a cmake build, so can we add the test to an existing component? (See also comment below regarding support.)

Reply:
That's what I wanted to do, but could you perhaps suggest an existing test to add this to?

Reply:
One of the `test_cmake_*` components, I guess. There's `test_default_cmake_gcc_asan`, `test_cmake_shared`, `test_cmake_out_of_source`. `test_cmake_as_package` and more. I don't know which one(s) makes the more sense: would pkg-config behave differently for those different builds?

Reply:
Sorry for the delay, I think `test_cmake_as_package` makes the most sense, moving it to there.